docs/guide/quick-start.md

```diff
@@ -84,11 +82,11 @@ class AuthHandler
                 } else {
                     $password = Yii::$app->security->generateRandomString(6);
                     $user = new User([
-                        'username' => $nickname,
-                        'github' => $nickname,
+                        'username' => $email,
```

Comment:
I don't think it is a good idea to use email as username despite it being a common practice nowadays:

1. It exposes one of the authentication factors.
2. In many cases you need to display username publicly. Email doesn't make sense for this case.